### DISCUSSION TITLE
缺氧更新

### DISCUSSION THREAD

**Discussion by xiaomaisuier** [Gamuxorg/bbs#288] [投递和请求游戏]
游戏名称：缺氧
游戏名称(英文)：oxygen not included
下载地址 :（如果有的话）：
版主：
问下，缺氧能更新一下吗？包括DLC

**Comment by shouhuanxiaoji** [comment]
magnet:?xt=urn:btih:730353DE1E2B5251C0574B53FBA43431598F60D1&tr=http%3A%2F%2Fbt.t-ru.org%2Fann%3Fmagnet&dn=Oxygen%20Not%20Included%20Bundle%20(498381)%20%2B%20OST%20%5Bamd64%5D%20%5BMulti%5D%20%5BSteam%5D%20%5BUnity3D%5D
使用bt客户端下载

**Comment by xiaomaisuier** [comment]
版主，你好。
上面文件用bt已经下载，得到一个后缀run文件，在终端下运行
sh ./oxygen_not_included_bundle_v498381.run
得到
YAD Simple Installer (050322z): Oxygen Not Included Bundle (498381)
安装yad后，运行yad ./oxygen_not_included_bundle_v498381.run
有个窗口弹出，只有确定和取消两个按钮，不论点哪个，就退出了，终端无任何信息。
问下，哪里要注意的，或我有什么地方没注意到的，帮忙指点下，谢谢！

**Comment by xiaomaisuier** [comment]
我在pixeldrain上下载了，谢谢！
不过，如果版主知道的话，也指点上面的问题，让我也学习下，谢谢！
